DedicatedServer/Scripts/Libs/Zrx/ModeLibs/Common/Collision/CylinderHitbox.Script.txt

```diff
@@ -1,14 +1,15 @@
+// #RequireContext CSmMode
```

Comment:
This whole file is not just a simple cylinder hitbox anymore. Please move changes to custom "Advanced Hitbox Library" which uses this library (if needed)